app/controllers/discourse_activity_pub/ap/actors_controller.rb

```diff
@@ -23,6 +23,7 @@ def ensure_can_access_actor
   end
 
   def ensure_can_access_actor_model
+    return true if @actor.ap.application?
```

Comment:
When my local ENV is marked as `login_required`, I can't follow a remote Discourse instance, and it looks like that is because the remote instance is failing the `ensure_can_access_actor` check (just above), this seems to return `false`: 

```
DiscourseActivityPub.publishing_enabled || @actor.ap.group?
```

When I change that to 

```
DiscourseActivityPub.publishing_enabled || @actor.ap.group? || @actor.ap.application?
```

the follow action works. Note though that then I can't unfollow the category, possibly because the `Undo` action also isn't permitted under a `login_required` scenario?

Reply:
Well caught. This will fix both: https://github.com/angusmcleod/discourse-activity-pub/commit/ae0978b85149ba83208165c1abbc7a44193a51b9